test/pylib/artifact_registry.py

```diff
@@ -28,22 +28,22 @@ def __init__(self) -> None:
 
     async def cleanup_before_exit(self) -> None:
         logging.info("Cleaning up before exit...")
-        for name in self.suite_artifacts:
-            for artifact in self.suite_artifacts[name]:
+        for suite_key in self.suite_artifacts:
```

Comment:
Better yet, since you're interested in the values and not the keys:
```python
for artifacts in self.suite_artifacts.values():
```
then use `artifacts` instead of `self.suite_artifacts[suite_key]`.

Reply:
done